cypress/typings/index.d.ts

```diff
@@ -14,11 +14,11 @@ declare global {
     // tslint:disable-next-line: interface-name
     interface Chainable {
 
-      open_source_panel: () => Chainable<JQuery<HTMLElement>>;
-      open_transform_panel: () => Chainable<JQuery<HTMLElement>>;
-      open_analytics_panel: () => Chainable<JQuery<HTMLElement>>;
-      open_sink_panel: () => Chainable<JQuery<HTMLElement>>;
-      open_condition_and_actions_panel: () => Chainable<JQuery<HTMLElement>>;
+      toggle_source_panel: () => Chainable<JQuery<HTMLElement>>;
```

Comment:
i'm not asking you to fix this here but we should remove this jquery type at some point.